README.md

```diff
@@ -11,5 +11,18 @@
 
 <p align="center"><img src="docs/images/Ui.png" width="100%"/></p>
 
-Welcome to InternHunter, a CLI-centric desktop application which aids university students in applying for tech
+
+Welcome to **InternHunter**, a CLI-centric desktop application which aids university students in applying for tech
 internships. It tracks and leverages on key metrics and information to create an optimal internship hunting experience.
+
+## Site Map
+- [User Guide](https://github.com/AY2021S1-CS2103T-T15-4/tp/blob/master/docs/UserGuide.md)
+- [Developer Guide](https://github.com/AY2021S1-CS2103T-T15-4/tp/blob/master/docs/DeveloperGuide.md)
+- [About Us](https://github.com/AY2021S1-CS2103T-T15-4/tp/blob/master/docs/AboutUs.md)
+ 
+ ## Acknowledgements
```

Comment:
Small issue, but should this line not begin with a whitespace? (Even though the syntax highlighting does not appear here on GitHub, I believe it still works to markdown the text in this case so shouldn't be an issue!)